tests/test_Admin.py

```diff
@@ -1194,3 +1195,34 @@ def test_delete_records():
 
     with pytest.raises(ValueError):
         a.delete_records([TopicPartition("test-topic1")])
+
+
+def test_elect_leaders():
+    a = AdminClient({"socket.timeout.ms": 10})
+
+    correct_topic_partition = TopicPartition("test-topic1", 0)
+    incorrect_partition = TopicPartition("test-topic1", -1)
+
+    correct_election_type = ElectionType.PREFERRED
+
+    # Incorrect Election Type
+    with pytest.raises(TypeError):
+        a.elect_leaders(None, [correct_topic_partition])
+
+    # Incorrect Partitions type
+    with pytest.raises(TypeError, match="Expected partitions to be a list, got 'str'"):
+        a.elect_leaders(correct_election_type, "1")
+
+    # Request-specific tests
```

Comment:
What does this mean?

Reply:
Changed it to Partition-specific tests.
It is specifically for partitions array validation checks